Amz.py

```diff
@@ -103,6 +104,19 @@ def give_away_type(driver):
                 return ga_type
 
 
+def confirm_address(driver):
+    # This will click the "confirm address" button for you
+    try:
+        driver.find_element(By.XPATH, '//input[@name="ShipMyPrize"]')
+        submit = WebDriverWait(driver, 20).until(
+			EC.element_to_be_clickable(
+				(By.XPATH, '//input[@name="ShipMyPrize"]'))
+		)
+        submit.click()
```

Comment:
You need to add `return True` here

Reply:
fixed in next commit: https://github.com/Penlo/Amazon-Giveaway-Bot/pull/12/commits/856a575e6d8d20a617488f7e5daf721838e571f4